doc/02-installation.md

```diff
@@ -266,7 +276,7 @@ systemctl start icinga2
 <!-- {% endif %} -->
 
 <!-- {% if rhel %} -->
-#### RHEL 8
+#### RHEL 8 or Later
```

Comment:
Not tested this (but you did), but not having to duplicate/C&P the section per release version is cool...